src/lib/entities/content-type.ts

```diff
@@ -140,23 +142,31 @@ class EditorInterfaces {
   }
 
   toAPI (): object {
-    let result: APIEditorInterfaceControl[] = []
+    let controls: APIEditorInterfaceControl[] = []
     forEach(this._controls, (c) => {
-      result.push({
+      controls.push({
         fieldId: c.fieldId,
         widgetId: c.widgetId,
         settings: c.settings
       })
     })
+
+    const result: {
+      controls: any,
+      sidebar?: any,
+      editor?: any
```

Comment:
it would be awesome if we could be more specific here instead of `any` I think we already have `APIEditorInterface...` types that we can probably use, I might be wrong though